app-management/guides/Editor Deep Link.md

```diff
@@ -0,0 +1,99 @@
+# Editor Deep Link
+
+
+## Introduction
```

Comment:
Our standard is to call this article "Introduction" in the menu and "About the Editor Deep Link API in the H1 header. No need for an "Introduction" H2

Reply:
done, in another edit